libs/ui/src/precinct_scanner_tally_report.tsx

```diff
@@ -135,22 +134,28 @@ export function PrecinctScannerTallyReport({
           <LogoMark />
           <Header>
             <h1>{reportTitle}</h1>
-            <h2>{electionTitle}</h2>
             <p>
-              {electionDate}, {election.county.name}, {election.state}
+              <strong>{electionTitle}:</strong> {electionDate},{' '}
+              {election.county.name}, {election.state}
+            </p>
```

Comment:
In cases where electionTitle includes the party name and is longer, I think this line could wrap. But in my opinion I don't think that's worth making any changes.

Reply:
Yes, wrapping here is fine. It will look better if it doesn't wrap, but it will still communicate what it needs to if the content does wrap.